### PR TITLE
feat(routes): add a simple status page

### PR DESCRIPTION
**Pull request by binarious**
Closes https://github.com/Sirimangalo/meditation-plus-angular/issues/489

### CODE REVIEW

**Comment by jiixing** [comment]
Should it access mongodb to check the connection, as current hanging issue is caused by the db connection going down

**Comment by olastor** [comment]
`mongoose.connection.readyState` should work (see https://stackoverflow.com/a/19606067), shouldn't it?

**Review by olastor — CHANGES_REQUESTED** [review]
Checking MongoDB connection would be useful. `mongoose.connection.readyState` should work (see https://stackoverflow.com/a/19606067), shouldn't it?

File: src/app/routes.js [hunk 1]
```diff
@@ -76,6 +76,11 @@ export default (app, router, passport, io) => {
   analyticsRoutes(app, router, admin);
   pushRoutes(app, router);
 
+  // Provide a simple status page
+  router.get('/status', (req, res) => {
```
Comment (olastor) [inline src/app/routes.js]: Do you have to be logged in to request this? Is there a frontend page or are you supposed to check this with i.e. `curl`?

**Review by olastor — CHANGES_REQUESTED** [review]
Can you please add a check for mongodb status as well?

File: src/app/routes.js (hunk 1 above)
Comment (jiixing) [reply src/app/routes.js]: I think the plan is to use 3rd party ping service like uptimerobot to go to /status to check the health of the db connection, and get email notification if the connection died

**Comment by binarious** [comment]
The mongoose check has been added.

**Review by olastor — APPROVED** [review]

**Pull request merged**